packages/algorand-js/test/integration/basic-teal.ts

```diff
@@ -60,7 +58,7 @@ describe("Algorand Smart Contracts", function () {
     // execute transaction (should fail is logic is incorrect)
     await expectTealErrorAsync(
       async () => await runtime.executeTx(invalidParams, 'incorrect-logic.teal', []),
-      ERRORS.TEAL.INVALID_STACK_ELEM
+      ERRORS.TEAL.LOGIC_REJECTION
```

Comment:
I think LOGIC_REJECTED is better. what do you think ?

Reply:
Let's use REJECTED_BY_LOGIC instead of these two. What do you say?